crates/forge/src/cheatcodes_hint_processor.rs

```diff
@@ -490,15 +477,23 @@ fn declare(
     );
     let tx = DeclareTransaction::new(
         starknet_api::transaction::DeclareTransaction::V2(declare_tx),
+        // TODO(#358)
+        TransactionHash::default(),
         contract_class,
     )
     .unwrap_or_else(|err| panic!("Unable to build transaction {err:?}"));
 
     let account_tx = AccountTransaction::Declare(tx);
     let block_context = build_block_context();
-    let _tx_result = account_tx
-        .execute(blockifier_state, &block_context)
-        .context("Failed to execute declare transaction")?;
+    let tx_result = account_tx.execute(blockifier_state, &block_context, true, true);
+
+    match tx_result {
+        Ok(_) => (),
+        Err(e) => {
+            return Err(anyhow!(format!("Failed to execute declare transaction:\n    {e}")).into())
+        }
+    };
```

Comment:
```suggestion
    match account_tx.execute(blockifier_state, &block_context, true, true) {
        Ok(_) => (),
        Err(e) => {
            return Err(anyhow!(format!("Failed to execute declare transaction:\n    {e}")).into())
        }
    };
```

`tx_result` isn't being used anywhere else, so we can at least combine these.